app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :username, presence: true, length: { in: 4..12 }, uniqueness: true
+
+  has_many :events
```

Comment:
* [ ] This is an unnecessary association. It doesn't have a belongs_to counterpart and it's unjustifiable logically. Kindly remove it.